toqito/matrices/fourier.py

```diff
@@ -2,7 +2,6 @@
 
 import numpy as np
 
-
```

Comment:
It's a PEP-8 thing, but the rules we abide by in the repo require two lines after imports (so putting back the removed line will fix the issue).

Reply:
Yes, exactly